machine/testdata/numa_meminfo

```diff
@@ -0,0 +1,29 @@
+Node 0 MemTotal:       33455420 kB
```

Comment:
I think huge pages parameters can be also useful(because we already have huge pages support under the k8s), regarding other parameters we can add them in the future.